sql/country_table.sql

```diff
@@ -0,0 +1,257 @@
+<?php
+
+include_once("../globals.php");
```

Comment:
Check this and see if it works from this directory.

Reply:
Checking....

Reply:
I had to change it to ../interface/globals.php

Reply:
and extension to .php because I am running the script manually

Reply:
Yep